package.json

```diff
@@ -19,6 +19,7 @@
     "postupdate": "yodoctor",
     "pretest": "xo",
     "test": "nyc mocha --timeout=10000",
+    "tabtab-install": "tabtab install --auto --name yo --completer yo-complete",
```

Comment:
If the user is intended to use `yo completion` command to install the completion mechanism, I think this not needed anymore.